src/components/Interpretations/common/getInterpretationAccess.js

```diff
@@ -1,6 +1,17 @@
+import { isArray } from 'lodash'
+
+// For backwards compatibility
+// accept both Set (from the old d2.currentUser object) and array
+const isSuperuser = (authorities = []) => {
+    if (isArray(authorities)) {
+        return authorities.includes('ALL')
+    }
+    return authorities.has('ALL')
+}
```

Comment:
It might be nice to have something like this, so we can easily add checks for other roles:
```suggestion
const hasAuthority = (authorities = [], authority) =>
    Array.isArray(authorities)
        ? authorities.includes(authority)
        : authorities.has(authority)

const isSuperuser = (authorities) => hasAuthority(authorities, 'ALL')
```
But not at all required